docs/known_limitations.md

```diff
@@ -0,0 +1,28 @@
+# Known Limitations
+
+## Purpose
+
+This set of documentation describes the data format GEMD.  This data concept is independent of any implementation of the format.  *However*, as the Citrine platform is assumed to be the primary point of interaction between this data model and most users, known limitations of this model are documented here, with appropriate links to this page provided elsewhere in this documentation.
+
+## Limitations
+
+* We cannot check for cycles in the material history.  
+
+    Material histories represent a chronology, so cycles in them represent time-travel (physically disallowed).  We do not have the ability to check that writing an object doesn't create a cycle in some material history.  
+
+* We have no support for Molecular Values (InChI, SMILES) or Molecular Bounds.  
+
+* We have no support for Composition Values (Nominal Composition, Empirical Formula), although Composition Bounds are supported.   
+
+* The values of name and labels fields of Ingredient Runs should be inherited from the associated Ingredient Specs, and thus be identically equal. However, due to implementation details, care must be taken around creating these objects:
```

Comment:
I don't think we need this here, since the specification here no longer even has `name` and `labels` on IngredientRuns.  This is documentation that specifically belongs in `taurus` (the python binding), IMO.